webapp/src/components/assetEditor/assetPalette.tsx

```diff
@@ -15,13 +18,20 @@ export interface AssetPaletteProps {
     onClose: (paletteChanged: boolean) => void;
 }
 
+const enum NameModal {
```

Comment:
Just curious: Why does this enum need to be const?

Reply:
It doesn't make a real difference here but const enums do compile slightly differently (i.e. you can't index into them with a value to get the enum member string it came from) https://www.typescriptlang.org/docs/handbook/enums.html#const-enums

Reply:
I guess it doesn't need to be. Should I drop the `const`?